frontend/src/pages/pipelines/global/pipelines/PipelinesPageHeaderActions.tsx

```diff
@@ -1,39 +1,72 @@
 import * as React from 'react';
-import { Dropdown, DropdownItem, DropdownSeparator, DropdownToggle } from '@patternfly/react-core';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownSeparator,
+  MenuToggle,
+  Tooltip,
+} from '@patternfly/react-core';
 import { DeleteServerModal, ViewServerModal } from '~/concepts/pipelines/context';
 
-const PipelinesPageHeaderActions: React.FC = () => {
+type PipelinesPageHeaderActionsProps = {
+  toggleElement: React.ReactNode;
+  variant: 'plain' | 'secondary';
+  isDisabled: boolean;
+};
+
+const PipelinesPageHeaderActions: React.FC<PipelinesPageHeaderActionsProps> = ({
+  toggleElement,
+  variant,
+  isDisabled,
+}) => {
   const [open, setOpen] = React.useState(false);
   const [deleteOpen, setDeleteOpen] = React.useState(false);
   const [viewOpen, setViewOpen] = React.useState(false);
+  const onToggleClick = () => {
+    setOpen(!open);
+  };
 
   return (
     <>
-      <Dropdown
-        removeFindDomNode
-        onSelect={() => setOpen(false)}
-        toggle={
-          <DropdownToggle toggleVariant="secondary" onToggle={() => setOpen(!open)}>
-            Pipeline server actions
-          </DropdownToggle>
+      <Tooltip
+        content={
+          isDisabled
+            ? 'To access the pipeline server actions, first create a pipeline server.'
+            : 'Pipeline Server Actions'
         }
-        isOpen={open}
-        position="right"
-        dropdownItems={[
-          <DropdownItem key="view-server-details" onClick={() => setViewOpen(true)}>
-            View pipeline server configuration
-          </DropdownItem>,
-          <DropdownSeparator key="separator" />,
-          <DropdownItem
-            onClick={() => {
-              setDeleteOpen(true);
-            }}
-            key="delete-server"
-          >
-            Delete pipeline server
-          </DropdownItem>,
-        ]}
-      />
+        position="bottom"
+      >
+        <Dropdown
+          removeFindDomNode
+          onSelect={() => setOpen(false)}
+          toggle={
+            <MenuToggle
+              aria-label="header-actions-toggle"
```

Comment:
This is not a good readable aria label. You need the label for the kebab but not for the text button. Doesn't hurt to be redundant though. May as well set it to `Pipeline Server Actions`